action.yml

```diff
@@ -38,7 +38,15 @@ inputs:
     description: "comment message"
     required: false
   path:
-    description: "path to file containing comment message"
+    description: "glob path to file(s) containing comment message"
+    required: false
+  ignore_empty:
+    description: "Indicates whether to ignore missing or empty messages"
+    default: "true"
```

Comment:
I don't know why I didn't notice this part before, but the default value is set to true. Most of the time you want to end up with an error and not skip it until you explicitly pass true. 
Also it cause of #829 and #825
We will change the default for this option in the next release. I guess you need to pass values explicitly in your CI. @samkio

Reply:
Ah yes it should be false sorry!

Reply:
No problem. :)